src/jdk.sctp/macosx/classes/sun/nio/ch/sctp/SctpChannelImpl.java

```diff
@@ -36,115 +36,115 @@
 import com.sun.nio.sctp.SctpChannel;
 import com.sun.nio.sctp.SctpSocketOption;
 
+import static sun.nio.ch.sctp.Util.sctpNotSupported;
+
 /**
  * Unimplemented.
  */
-public class SctpChannelImpl extends SctpChannel
-{
-    private static final String message = "SCTP not supported on this platform";
+public class SctpChannelImpl extends SctpChannel {
```

Comment:
Going a bit beyond just updating syntax; but that's a different PR...
There could be more sharing of the unsupported implementation if SctpChannel was not abstract and its method bodies threw the exception.  There would less duplication for unsupported platforms.

Reply:
> There could be more sharing of the unsupported implementation if SctpChannel was not abstract and its method bodies threw the exception. There would less duplication for unsupported platforms.

I think this is accident of history. As I recall, the "unsupported implementation" was Windows only in JDK 6 but it has since been copied. It could be cleaned up.

Reply:
Created https://bugs.openjdk.org/browse/JDK-8297822?filter=-2 and will move changes for this to a separate PR.